Chore: (Docs Intro to Storybook) Intro to Storybook adjustments (#456)
chromaui/learnstorybook.com: content/intro-to-storybook/react/en/simple-component.md

## Config

> We'll need to make a couple of changes to Storybook's configuration files so it notices not only our recently created stories, but also allow us to use the application's CSS file (located in `src/index.css`).

Reviewer: ```suggestion
We'll need to make a couple of changes to Storybook's configuration files so it notices our recently created stories and allows us to use the application's CSS file (located in `src/index.css`).
```

Author: Fixed with the latest commit. Marking this as resolved
